minkyu/app.js

```diff
@@ -36,10 +39,40 @@ app.use(express.json());
 app.use(cors());
 app.use(morgan('tiny'));
 
-app.get("/ping", (req,res) =>{
-    res.json({message : "pong"});
+app.get("/pings", (req,res) =>{
+    res.status(201).json({"message" : "pong"});
 });
 
+app.post("/users", async(req, res) =>{
+    const { name, email, password } = req.body;
+
+    await database.query(
+        `INSERT INTO users(
+            name,
+            email,
+            password
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [ name, email, password ]
+    );
+
+    res.status(201).json({ message : "successfully created" });
+})
+
+app.post("/posting", async(req, res) => {
+    const {title, content, user_id} = req.body;
+
+    await database.query(
+        `INSERT INTO posts(
+            title,
+            content,
+            user_id
+        ) VALUES (?, ?, ?)
+        `,
+        [ title, content, user_id ]
```

Comment:
```suggestion
        [ title, content, userId ]
```
위의 63번째 line에서 구조분해 할당 할 때에 userId로 수정되었으니 이에 맞춰 수정되어야겠습니다!